### DISCUSSION TITLE
How to enable binary logging in planetscale?

### DISCUSSION THREAD

**Discussion by yichulau** [planetscale/discussion#418] [FAQs] [answered]
Hi guys, i have a question how can i enable binary logging in planetscale?

**Comment by jgreet** [comment, marked as answer]
Hi @yichulau, unfortunately we can't expose MySQL binlogs from PlanetScale databases currently.

We do have Airbyte and Stitch integrations if you need to sync your data to another location:

https://planetscale.com/blog/extract-load-and-transform-your-data-with-planetscale-connect
